DataManagementSystem/Utilities/MetaQuery.py

```diff
@@ -0,0 +1,208 @@
+########################################################################
```

Comment:
I guess this MetaQuery class replaces the one in DataManagementSystem/Client/MetaQuery.py. If it is the case, the original one should be removed